dist/init/linux-systemd/caddy.service

```diff
@@ -5,18 +5,32 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
+; Only allow 1 restart with 10 seconds so that caddy doesn't get locked out by PAM
 Restart=on-failure
-StartLimitInterval=86400
-StartLimitBurst=5
+StartLimitInterval=10
+StartLimitBurst=1
```

Comment:
This *hunk* is out of scope, which is usually grounds to reject it.
But I agree, the previous settings are ridiculous, so I am okay with the change.

Keep in mind that these influence not only startup, but restarts (automatic, manual…) too.  
To the argument, nobody does two (or more) meaningful restarts within 10 seconds: You can have *systemd* watch files like Caddy's configuration file (by a `*.path` one) and trigger restarts by that. So if someone currently edits the config and saves twice… The latest edit will be discarded. Such are the side-effects. (Okay to neglect them here.)

If you like, change `Restart` to `on-abnormal`. I don't see any reason to try a failed start again. If something like an IP address were the cause for the initial failure, then there's a configuration error somewhere else.